modules/docs/mdx/12.0-UPGRADE-GUIDE.mdx

```diff
@@ -132,6 +135,18 @@ The following components are affected:
   prop now accepts `"dark" | "light"`
 - The `as` prop now accepts any element, not just `div`.
 
+### Form Field
+
+Previously, the FormField component’s `orientation` prop only accepted the values `vertical` and
+`horizontal` to align the label.
+
+We’ve updated the `orientation` prop on the FormField component to accept the following values:
+`vertical`, `horizontalStart` and `horizontalEnd`. If you previously had set `orientation` to
+`horizontal` a codemod will update this value to `orientation="horizontalStart"`.
+
+> **Note**: The horizontal alignment of start and end are relative to its container, meaning start
+> and end match the flex property of `flex-start` and `flex-end`.
```

Comment:
This can be worded differently but, this is a thought. 

```suggestion
> **Note**: The horizontal alignment of start and end are relative to its container, meaning start
> and end match the flex property of `flex-start` and `flex-end`. This is especially applicable for moving between RTL (right-to-left) and LTR (left-to-right) languages. 
```